linux_docker_resources/Dockerfile-CentOS

```
@@ -15,6 +15,7 @@ RUN yum -y copr enable cottsay/devtoolset-8-make-nonblocking
 RUN yum install \
   @buildsys-build \
   asio-devel \
+  bullet-dev \
```

This is still not right.
```suggestion
  bullet-devel \
```